front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorForm.tsx

```diff
@@ -37,6 +41,30 @@ type RollingStockParametersProps = {
   isAdding?: boolean;
 };
 
+export function modifyRollingStockElectricalValues(
```

Comment:
Maybe it would be better to move this function to a helper file ?

Reply:
I don’t think it’s ever going to be used anywhere else. it’s just for this component